blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.models import entity_features
+from mindmeld.markup import process_markup
+from mindmeld.core import FormEntity
+import random
+import json
+
+app = Application(__name__)
+
+__all__ = ['app']
+
+user_data = {}
+
+sample_users = {
+    "johndoe123" : 0,
+    "larry_l12" : 1,
+    "splashbro30" : 2,
+}
+
+def pull_data(request):
+    with open('banking_blueprint/data/sample_user_data.json') as f:
+        global user_data 
+        data = json.load(f)
+        if request.context.get('user_name'):
+            user_name = request.context.get('user_name')
+            user = sample_users[user_name]
+        else:
+            user = random.choice([0, 1, 2])
+
+        user_data = data[user]
+        print(user_data)
+
+def get(key):
+    print(key)
+    return user_data[key]
+
+def put(key, value):
+    global user_data
+    user_data[key] = value
+
+@app.handle(intent='greet')
+def greet(request, responder):
+    if not user_data: 
```

Comment:
If the user_data is already set in one turn, can it be changed during the conversation if this defined?